test/rules/std_L016_L36_combo.py

```diff
@@ -0,0 +1,31 @@
+"""Tests the combination of L016 (no long lines) and L036 (single selects should eb on SELECT line)."""
+
+import sqlfluff
+
+
+def test__rules__std_L016_L036_long_line_lint():
```

Comment:
These tests are good. Noting that another possibility would be to add a test to `test/fixtures/linter/autofix/`. (Personally, I think there are slightly more styles of tests than I can keep track of. Could be cool someday to combine some of these and/or provide docs telling you the preferred way of testing various kinds of changes.) 🤯